druid/src/widget/widget_ext.rs

```diff
@@ -125,8 +125,38 @@ pub trait WidgetExt<T: Data>: Widget<T> + Sized + 'static {
 
     /// Wrap this widget in a [`LensWrap`] widget for the provided [`Lens`].
     ///
+    /// Lenses are useful whenever a widget only needs access to a subfield of a larger struct.
```

Comment:
I think this is a bit narrow; lenses can be useful for accessing the interior of things other than structs as well. Perhaps "access to part of a larger value"?

Reply:
You're right. I now included both statements, as I think the concept is hard to get at first (for me it was) I tried to be as concrete here as possible.